.github/workflows/build.yml

```diff
@@ -91,6 +95,8 @@ jobs:
         with:
           args: 
              "${{ steps.imagemagick.outputs.version }}"
+      - name: Test
+        run: "dpkg-deb -f ${{ github.workspace }}/imagemagick_${{ steps.imagemagick.outputs.version }}_amd64.deb Depends | grep -qEv 'libcdt|libcgraph|libgvc'"
```

Comment:
As you are testing `Depends` only you can also add libgs package (or whatever its name on debian) in the list